script/Ciel/Day 9/09_01_CIEL09_2B.txt

```diff
@@ -1391,109 +1586,133 @@ I don't give a damn whether she was being serious or just trying to rile me up.
 [sha:d01136379096ad8f457455cf8bf12aac76236456]{
 -- Page 252, Offset 22786.
 -- 先輩にとってアルクェイドは倒すだけの吸血鬼で、そんなものを信じる俺は愚か者だと思っている。
-To Senpai, Arcueid is nothing more than a vampire that must be defeated, and she thinks I'm a fool for trusting her.
+It's clear that either way,
+ Senpai thinks Arcueid is nothing more than a vampire to be defeated,
+ and that I'm a fool for trusting her.
 }
 [sha:49995e4ab7f9806145ef55e1a2452aac7ad17d18]{
 -- Page 253, Offset 22787.
 -- 「俺は騙されてなんかないっ！　だいたい、アイツが嘘なんかつけるもんか、先輩みたいに器用じゃないんだから！
-"I'm not being deceived! Arcueid doesn't lie anyway, she isn't cunning like you!
+// 2 dumb 2 lie
+"I'm not being deceived!
+ As if she's even capable of lying in the first place.
+ She's not a snake like you, Senpai!
```

Comment:
Shiki isn't actually being that caustic here, so I'd refrain from making him sound like it.

>I'm not being deceived!
> As if she's even capable of lying in the first place.
> She's not as tricky as you are, Senpai!